File: src/main/java/net/atos/zac/app/zoeken/model/AbstractRESTZoekObject.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.zoeken.model;

import net.atos.zac.zoeken.model.index.ZoekObjectType;

public abstract class AbstractRESTZoekObject {

  public String id;

  public ZoekObjectType type;

  public String identificatie;
}

File: src/main/java/net/atos/zac/app/zoeken/model/AbstractRestZoekObject.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.zoeken.model

import net.atos.zac.zoeken.model.zoekobject.ZoekObjectType
import nl.info.zac.util.NoArgConstructor

@NoArgConstructor
abstract class AbstractRestZoekObject {
    var id: String? = null

    var type: ZoekObjectType? = null

    var identificatie: String? = null
}
